backend/index.js

```diff
@@ -14,9 +15,9 @@ app.use(bodyParser.json());
 mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true, useUnifiedTopology: true });
 var db = mongoose.connection;
 
-const Profile = require("./models/profile.js");
-const Likes = require("./models/Likes.js");
-const User = require("./models/User.js");
+const Profilerouter = require("./routes/profiles.js");
```

Comment:
Now that it's working, i'm gonna set up a separate routes.js file to clean this up a bit :)